src/term/mod.rs

```diff
@@ -2008,7 +2008,7 @@ impl ansi::Handler for Term {
             ansi::Mode::DECCOLM => self.deccolm(),
             ansi::Mode::Insert => self.mode.insert(mode::TermMode::INSERT), // heh
             _ => {
-                trace!(".. ignoring set_mode");
+                trace!("... ignoring set_mode");
```

Comment:
Unsure what the `...` is here for at all? Maybe this should just be removed? Same for the `ignoring unset_mode` trace.

Reply:
I think it's implying that the above `set_mode: ...` was a noop for that mode.

Reply:
Yeah, I'm just unsure if this is a structure we should keep or rework. Looking at it again, it should be fine for tasks which reference a previous log message, though it should be made absolutely clear which log message is referenced, since it's not guaranteed that there's no other message in between.

I'd probably go for changing this to `... ignoring attribute`.